bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/AbstractTreeViewer.java

```diff
@@ -102,8 +112,24 @@ public abstract class AbstractTreeViewer extends ColumnViewer {
 	private int expandToLevel = 0;
 
 	/**
-	 * Indicates if filters should be checked to determine expandability of
-	 * a tree node.
+	 * How many levels to autoexpand in a path of consecutive single children. 0
+	 * will disable the feature, while a negative value (<code>ALL_LEVELS</code>)
```

Comment:
You should never use the value of a constant for documentation, as that value may change (that's one of the reason why it was defined as a constant).
```suggestion
	 * will disable the feature, while <code>ALL_LEVELS</code>
```